### ISSUE TITLE
genesis.bootstrap fails if no pkg AND exclude_pkgs (which can't be a string)

### ISSUE THREAD

**Issue by aogier** [saltstack/salt#43101]
### Description of Issue/Question

`genesis.bootstrap` module fails if no exclude_pkgs is specified:

```
# salt-call --local genesis.bootstrap deb /tmp/cane arch=amd64 flavor=xenial repo_url=http://archive.ubuntu.com/ubuntu
[ERROR   ] Command '['debootstrap', '--foreign', '--arch', 'amd64', '--include', '--exclude', 'xenial', '/tmp/cane', 'http://archive.ubuntu.com/ubuntu']' failed with return code: 1
[ERROR   ] output: I: Retrieving InRelease 
I: Checking Release signature
I: Valid Release signature (key id 790BC7277767219C42C86F933B4FE6ACC0B21F32)
I: Retrieving Packages 
I: Validating Packages 
...
<<long deboostrap output>>
...
E: Couldn't download packages: --exclude

Passed invalid arguments: 'NoneType' object is not iterable.
```

As reported by stack trace, resulting command line is indeed: 
```
/bin/sh /usr/sbin/debootstrap --foreign --arch amd64 --include --exclude xenial /tmp/cane http://archive.ubuntu.com/ubuntu
```

this is obviously a wrong invocation for --include/--exclude flags. Also, it's not possible to pass them via cmdline eg.

```
# salt-call --local genesis.bootstrap deb /tmp/cane arch=amd64 flavor=xenial repo_url=http://archive.ubuntu.com/ubuntu exclude_pkgs=vim,vim pkgs=bash

Passed invalid arguments: can only concatenate list (not "str") to list.
```

Well since I need it for write tests for #43086 I'll try to fix this before :)

Thank you ciao

### Versions Report
```
Salt Version:
           Salt: 2017.7.1
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Nov 19 2016, 06:48:10)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
         locale: ANSI_X3.4-1968
        machine: x86_64
        release: 4.4.0-92-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by gtmanfred** [comment]
Thanks for the PR to fix this @aogier 

Daniel

**Comment by aogier** [comment]
closed in f16b724
